packages/devtools_app/lib/src/inspector/flutter/inspector_service_flutter_extension.dart

```diff
@@ -50,9 +52,11 @@ extension InspectorFlutterService on ObjectGroup {
         '  dynamic object = WidgetInspectorService.instance.toObject("${ref?.id}");'
         '  if (object == null) return null;'
         '  final render = object.renderObject;'
-        '  final FlexParentData parentData = render.parentData;'
-        '  parentData.flex = $flexFactor;'
-        '  render.markNeedsLayout();'
+        '  final parentData = render.parentData;'
+        '  if (parentData is FlexParentData) {'
+        '    parentData.flex = $flexFactor;'
+        '    render.markNeedsLayout();'
+        '  }'
```

Comment:
nit: maybe return false if you cannot set the properties?

Reply:
currently we are not storing/doing anything with the result from this call though, should we show some error in the scaffold or something?